Topic1_Arrays/Day2207/1.txt

```diff
@@ -0,0 +1,14 @@
+var twoSum = function(nums, target) {
+    let result = [];
+    let lengthNums = nums.length;
```

Comment:
Em nên có thêm phần đánh giá độ phức tạp của thuật toán
Tìm thử thêm hướng làm với HashMap để giảm độ phức tạp